helm-charts/konk-operator/templates/deployment.yaml

```diff
@@ -33,18 +32,6 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-          ports:
-            - name: http
-              containerPort: 80
-              protocol: TCP
-          livenessProbe:
-            httpGet:
-              path: /
-              port: http
-          readinessProbe:
-            httpGet:
-              path: /
-              port: http
```

Comment:
We need a ticket to reintroduce operational probes. There are consequences when an apiservice is unavailable ie. deleting namespaces will be blocked. We'd want to know if konk-operator was flapping or 503.

Reply:
The task here will be figuring out if the helm-operator provided by operator-sdk has a health endpoint that can be used for probes. The generated code doesn't reference one.